spring-redisgraph-configuration/src/main/java/com/redislabs/springredisgraph/RedisGraphConfiguration.java

```diff
@@ -46,4 +53,10 @@ private int port() {
         return port;
     }
 
+    private String password() {
+        if (password == null) {
+            return props.getHost();
```

Comment:
shouldn't it suppose to be`props.getPassword`?

is there a default value in case of no password?

Reply:
changed to getPassword